### PR TITLE
Improving MariaDB chart NOTES.txt

### PR DESCRIPTION
**Pull request by juan131**
**What this PR does / why we need it**:

Improves the Notes displayed to the user when installing the MariaDB Chart:

- Provides information about how to connect to MariaDB from outside the cluster.
- Adds a Security Warning when using LoadBalancer as Service Type.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/mariadb/templates/NOTES.txt [hunk 1]
```diff
@@ -1,15 +1,34 @@
+{{- if contains .Values.service.type "LoadBalancer" }}
+{{- if not .Values.usePassword }}
+-------------------------------------------------------------------------------
+ WARNING
+
+    By specifying "service.type=LoadBalancer" and "userPassword=false" you have
+    most  likely exposed the MariaDB service externally withoy any authentication
```
Comment (sameersbn) [inline stable/mariadb/templates/NOTES.txt]: typo `withouy`

File: stable/mariadb/templates/NOTES.txt [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
+    mechanism.
+
+    For security reasons, we strongly suggest that you switch to "ClusterIP" or
+    "NodePort". As alternative, you can also switch to "userPassword=true"
```
Comment (sameersbn) [inline stable/mariadb/templates/NOTES.txt]: typo `userPassword`

File: stable/mariadb/templates/NOTES.txt [hunk 3]
```diff
@@ -21,3 +40,26 @@ To connect to your database:
...
+  NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+        Watch the status with: 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "mariadb.fullname" . }}'
+
+    export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "mariadb.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    mysql -h $SERVICE_IP -P {{ .Values.service.nodePort }} -uroot {{- if .Values.usePassword }} -p$MARIADB_ROOT_PASSWORD{{ end }}
+
+{{- else if contains "ClusterIP" .Values.service.type }}
+
+    export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "mariadb.name" . }}" -o jsonpath="{.items[0].metadata.name}")
+    kubectl port-forward $POD_NAME 3306:3306 &
```
Comment (sameersbn) [inline stable/mariadb/templates/NOTES.txt]: add `--namespace {{ .Release.Namespace }}` to the port-forward command

**Review by sameersbn — CHANGES_REQUESTED** [review]

File: stable/mariadb/templates/NOTES.txt [hunk 4]
```diff
@@ -1,15 +1,34 @@
+{{- if contains .Values.service.type "LoadBalancer" }}
+{{- if not .Values.usePassword }}
+-------------------------------------------------------------------------------
+ WARNING
+
+    By specifying "service.type=LoadBalancer" and "userPassword=false" you have
```
Comment (tompizmor) [inline stable/mariadb/templates/NOTES.txt]: `usePassword` here too

Comment (juan131) [reply stable/mariadb/templates/NOTES.txt]: 👍

File: stable/mariadb/templates/NOTES.txt (hunk 1 above)
Comment (juan131) [reply stable/mariadb/templates/NOTES.txt]: 👍

File: stable/mariadb/templates/NOTES.txt (hunk 3 above)
Comment (juan131) [reply stable/mariadb/templates/NOTES.txt]: 👍

**Review by sameersbn — APPROVED** [review]

**Pull request merged**